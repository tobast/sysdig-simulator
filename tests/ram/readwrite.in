8
0000000010000000011100110
0000000010000100010000000
0000000000000000000000000
0000100000000000000000000
0000000010000000000000011
0000000000000000000000000
0000001010000001011111111
0000001000000000000000000
