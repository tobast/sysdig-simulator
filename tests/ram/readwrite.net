INPUT rAddr, doWrite, wAddr, data
OUTPUT ramVal
VAR
	rAddr:8, wAddr:8, data:8, doWrite, ramVal:8
IN
ramVal = RAM 8 8 rAddr doWrite wAddr data
