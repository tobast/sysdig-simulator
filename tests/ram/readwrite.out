00000000
11100110
11100110
10000000
11100110
00000011
00000000
11111111
